custom-transform/app.R

```diff
@@ -0,0 +1,137 @@
+library(teal)
+
+my_transformers <- list(
+  teal_transform_module(
+    label = "Keep first 6 from IRIS",
```

Comment:
This is based on input so it can't be fixed to `6`
```suggestion
    label = "Keep first n-observations from IRIS",
```